src/redux/selectors/index.js

```diff
@@ -140,8 +141,8 @@ export const timelineIntervalsSelector = createSelector(
     // const maxDate = timelineItemsInRange[timelineItemsInRange.length - 1]?.timelineDate;
 
     if (minDate && maxDate && timelineItemsInRange.length) {
-      const numDays = differenceInDays(maxDate, minDate);
-      const intervalCount = Math.min(numDays, MAX_INTERVAL_COUNT) || 1; // cannot be 0
+      const numDays = Math.max(differenceInDays(maxDate, minDate), 1);
+      const intervalCount = Math.min(numDays, MAX_INTERVAL_COUNT); // cannot be 0
```

Comment:
^This is a fix for when the date-pickers are set to a one day range:  it will be the beginning of day to end-of-day, which `differenceInDays` interprets as 
"0"  (less than one full day)